_data/mentors.yml

```
@@ -98,6 +103,8 @@
       - Backend Developer
       - Distributed Systems
       - Fullstack Developer
+      - Distriibuted Systems
```

```suggestion
      - QA
```
You can also add as suggestion and we can fix direct by the comment 😄